### ISSUE TITLE
ProPerty '__PrismaProxy' Doesn't exist

### ISSUE THREAD

**Issue by CRIIPI11** [prisma/react-native-prisma#30]
When I try to intialize the prisma client on my app, I get the following warning not letting me run queries
![D83BC388-273C-4081-9C24-87A86E64F331](https://github.com/prisma/react-native-prisma/assets/60235905/f72b297b-80aa-4237-a1b1-0784b06e0d12)

**Comment by Kureev** [comment]
Same thing:

```
"@prisma/client": "^5.16.2",
"@prisma/react-native": "^5.16.2",
"react": "18.2.0",
"react-dom": "18.2.0",
"react-native": "0.74.2",
...
```

**Comment by Kureev** [comment]
Small update:
- I've tried multiple versions (various dev versions), no difference
- Tried Android, same issue
- Tried both: turbo modules and old bridge arch

**Comment by CRIIPI11** [comment]
@Kureev any luck yet ?

**Comment by Kureev** [comment]
@CRIIPI11 Not really. I spent some time looking into this, but it doesn't look like an issue "on the surface", so I'm afraid we will have to wait for an official "fix". 

Offtopic: I have a proof of concept for the library similar to react-native-prisma (expo-sqlite + *.prisma schema that generates hooks etc just like Prisma, but it doesn't fully support migrations and there is likely tons of bugs), so I might release something in the future if this lib won't progress to the stable version within reasonable timeframe. 

I'm a strong believer in the concept and I absolutely love Prisma, but I have my own business use cases and unfortunately I can't afford to wait for unknown amount of time to plug in the database. 

Nevertheless, I hope this library (and this issue in particular) will receive the love it deserves ❤️ . I might drop a link to the lib I'm building once it is more stable and tested.

**Comment by Kureev** [comment]
so just in the case you are interested, [there is a draft](https://github.com/Kureev/veloria) of the lib that will cover some basic functionality. Many things are missing atm, but it should be good enough for dev preview.

**Comment by sorenbs** [comment]
Could this be related to the issue and fix described here: https://github.com/prisma/react-native-prisma/issues/16#issuecomment-2212574178 ?

**Comment by CRIIPI11** [comment]
@sorenbs Did you try the solution posted in that issue ?

**Comment by Kureev** [comment]
I believe I've been running it with `prisma.schema` in the root from the very beginning, but will give it another attempt today if I won't forget.

**Comment by piscopancer** [comment]
RN prisma does not work in Expo Go. You need to migrate prisma, prebuild clean and then run a development build via `npx expo start --dev-client`
